.github/workflows/test-on-ryot.yml

```diff
@@ -21,19 +21,19 @@
 #   * udev rules that map all BOARD to /dev/riot/tty-$(BOARD), see
 #     http://riot-os.org/api/advanced-build-system-tricks.html#multiple-boards-udev
 #   * RIOT_MAKEFILES_GLOBAL_PRE that sets PORT and DEBUG_ADAPTER_ID for each
-#     BOARD http://riot-os.org/api/advanced-build-system-tricks.html#multiple-boards-udev
+#     BOARD
 #   * Static list of connected boards or make target listing them in json format
-#     so that fromJSON can be used, see:
-#     https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#fromjson
-#
+#     so that fromJSON can be used.
+#       - static list: "boards: [samr21-xpro, nucleo-l152re, ..]""
```

Comment:
This does not seem like legal JSON. From the link below, I assume it should be

```suggestion
#       - static list: ["samr21-xpro", "nucleo-l152re", ...]
```

Reply:
The static list does not use `fromJSON`, `fromJSON` is used when loading a list dynamically. So maybe its not clear, for the job there is two ways of setting the matrix:

```
    strategy:
      matrix:
        board: ${{ fromJson(needs.connected_boards.outputs.boards) }}
```

and 

```
    strategy:
      matrix:
        board: [samr21-xpro, nucleo-l152re]
```

The first having the advantage that sometimes I plug/unplug BOARDs so you won't have a failing test when a BOARD is removed, and I don't have to edit the file when adding BOARDs to the ci.

The second is simpler and if you don't expect to remove/add boards often is perfectly fine.

Reply:
I think the first is simply not of any interest for someone adding their own RYOT machine to this workflow. So maybe it should just not be mentioned here or in this context.

Reply:
Ok so remove mentions to `fromJson`?

Reply:
No, the mention of the static list.

Reply:
Since it won't be usable when this is extended anyway. Seems more like a debug solution to me, that should be clear if you see how matrix is set in the deployed solution.

Reply:
Then you meant second

Reply:
I meant the first option in the bullet list in your doc ;-)

Reply:
Ahhh then we are both right :D